src/redux/epics/upload-epic.js

```diff
@@ -82,7 +84,9 @@ const pollUploadProgress = (action$, store) => {
           backIdx: addresses.length - 1
         })
       ),
-      Observable.interval(3000)
+      Observable.interval(
+        BUNDLE_SIZE * SECONDS_PER_CHUNK / NUM_BROKER_CHANNELS * 1000
```

Comment:
Tiny optimization: Make this a `const` at the top of the file so we don't have to recalculate this inside the epic.